src/FnacApiClient/Type/LogisticTypeId.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+namespace FnacApiClient\Type;
+
+/**
+ * LogisticTypeId
+ */
+class LogisticTypeId extends AbstractType
```

Comment:
Is it used somewhere?

Reply:
https://github.com/productsupcom/fnacapiclient/pull/1/files#diff-352c86717bf9afab99ab1755e3a1739eR536 
I just wanted to stick to fnac way of writing this part. This way if we were to modify this lib we would have a consistent lame code base.